src/component/AddressBody.jsx

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import { useState } from "react";
+import { UseAddressContext } from "../context/address-context";
+import "../pages/auth.css"
+import { AddressForm } from "./AddressForm";
+export function AddressBody() {
+
+
+
+    const [addNew, setAddNew] = useState("none")
+    const {pincodeDes} = UseAddressContext()
+    const {addAddress, address , removeAddress} = UseAddressContext();
+    const ListOfAddress = () =>{
```

Comment:
No need to check for [], undefined
Just check it for value it returns when there's no data.